cloudinit/config/schema.py

```diff
@@ -937,9 +938,14 @@ def validate_cloudconfig_file(
             )
             return False
     try:
-        validate_cloudconfig_schema(
+        if not validate_cloudconfig_schema(
             cloudconfig, schema, strict=True, log_deprecations=False
-        )
+        ):
+            print(
+                "Skipping network-config schema validation."
```

Comment:
Because we don't know if we are skipping network-config or cloud-config
```suggestion
                f"Skipping {schema_type}schema validation."
```